lowRISC/opentitan
# [DO NOT MERGE] Test rules_python with local package

The purpose of this PR is to see if we can define local package using `pyproject.toml` and let `rules_python` use them as usual.

How to test:
- try it with:
```console
./bazelisk.sh run @ot_python_deps_hello//:rules_python_wheel_entry_point_hello
./bazelisk.sh run @ot_python_deps_hello//:rules_python_wheel_entry_point_bye
```

I added a patch so that rules_python watchs the local package's source code. You can try it by changing `hello.py` and see that if you re-run the commands above, the package is reinstall and the change is picked up.
